examples/ensureType-function/ensureType_function.bal

```diff
@@ -0,0 +1,15 @@
+import ballerina/io;
+
+function demo(anydata v) returns float|error {
+    // `v` is cast to the float.
```

Comment:
I think we can rephrase this and specifically mention that it returns an error instead of panicking unlike with casts.